source/dbo/Stored Procedures/Power BI Reporting/uspGenerateRecordOutcome.sql

```diff
@@ -48,7 +48,15 @@ BEGIN TRY
 			JOIN [$(NTBS)].ReferenceData.TbService tbs ON tbs.Code = te.TbServiceCode
 		WHERE TreatmentEventType = 'TransferOut';
 
-	WITH NotifyingServiceAndCode AS 
+	WITH NotifyingServiceAndCodeFromInitialEvent AS
+	(SELECT DISTINCT NotificationId,
```

Comment:
I think the `DISTINCT` here adds a false sense of security - if the notification did somehow end up with more than one treatment start/diagnosis made event then it's also possible that those events would have different TB services. I think it's better not to have the `DISTINCT` call and maybe comment saying each notification will only have one such event.